### ISSUE TITLE
IMAGEN v2 project: minutes

### ISSUE THREAD

**Issue by RoshanRane** [RoshanRane/ML_for_IMAGEN#3]
**Meeting minutes: Date:**  _30 June 2022_<br>
Attended: Roshan, Milena and Kerstin<br>
1. ~Get other variables of interest from Milena and run the analysis on them -> July 1st week~
2. ~Why didnt controlling for ESPAD-Alc reduce it so much -> use age 22 instead of 18~
3. ~Control for binge variable itself or a noisy variant of it?~
4. ~Should I instead try testing it on my new 3D tool by training a neural network on the data?~
5. ~We could not even predict personality vars from sMRI? Add additional plot to show this vs binge drinking~
6. ~color the variables by categories~
7. ~check smoking var used in Whelan paper~

**Pointers for Milena after first iteration of main analysis:** _Date: 12 July 2022_<br>
* ~Analysis 0 is where I preprocess all the missing confounds. You should check the distribution of these variables to see that the median binary threshold makes sense for them~  -->  mean is a better threshold
* ~Ensure that the table Confounds_list_Milena.xlsx has no '???' or empty cells in columns ['colname', 'questionnaire', 'impute_missing_with','binarizing_threshold','category']~ --> done
* ~In Analysis 1 I prepare the h5files. I have shown an example for the Imaging-at-BL-onset0 h5file. It would be helpful if you take a look at all the output msgs and distributions once, just in case I missed something.~ --> done

---

**Meeting minutes: Date:**  _05 Aug 2022_<br>
Attended: Roshan, Milena and Kerstin<br>
1. Exclude FU3 analysis from the paper. It can be confusing and isn't so useful from a clinical perspective.
2. ~Run an analysis with the BL h5 file.~
3. Run permutation tests for BL-causal0 and BL-causal1 splits.
4. Summarise the results from BL, BL causal0, and BL causal1 in a single plot to check robustness of different variables.
5. Show permutation results on the plot.
6. Categorize the covariates by interesting domains (Ex: life stressors, impulsive behaviors) for the paper discussion.

---
**Meeting minutes: Date:**  _07 Sept 2022_<br>
Attended: Roshan and Kerstin<br>
Discussion and decisions made with Kerstin on 7 sept 22: (Kerstin is on maternal leave from 22nd sept. So we should finish the draft before that and get it reviewed)

__Major pts (1) 3 data subsets analyses did not add much value to the paper:__

Repeating the analysis on 3 data subsets did not add much value to the results and is only serving to confuse the readers. Furthermore, the minor heterogeneity in the results (explorative + gen) across the 3 subsets would make us vulnerable to reviewer criticisms.

+ __Solution (a1):__ We can write this paper with a single data subset *BL-binge-0*, which is the most clinically relevant one.
+ __Solution (a2):__ However, the *BL-binge-0* is the least robust due it its sample size. The permutation results also show this (see results/results-permutations.jpg on GitHub). A good compromise would be to use the *BL-binge-0* and *BL-binge-1* during the explorative stage and report gen stage results only on *BL-binge-0*. We can motivate this by saying - "We repeated the explorative analysis also on *BL-binge-1* data, under the rationale that if there exist potential explanatory variables that largely influence the accuracy on *BL-binge-0* data subset, then these variables should also appear on *BL-binge-1*, albeit to a smaller extend. This was an additional check we added for the robustness of the findings."
Therefore, we will only need to explain this briefly in the Methods->analysis section once and nowhere else in the paper. If reviewers request, we can also show the gen results on  *BL-binge-1* in supplements

__Major pt (2) We can completely exclude GB and only show results on SVM-rbf.__
This would simplify the results to a large extent. In the introduction, we can already establish that we chose SVM for this analysis as it performs the best among all 4 models and in (6) the brain region importance (visualization) was derived only for SVM (see line 316 in medarxiv preprint)

__Minor pt (3) do not call the variables we investigate as confounds but something like 'potential' explanatory variables.__
 Only in the Methods chapter, we can clarify that we formulate the research question as a confound correction analysis.

__Minor pt (4) Cannot use permutation threshold in the exploratory stage.__

We realized that permutation tests cannot be used in the exploratory stage as the model accuracies are not independent estimates when derived using n-fold CV (this was pointed out by an internal reviewer and hence we had to show p-values only for gen stage in the final paper). Furthermore, there is a lack of clear motivation as to why we do the analysis in 2 stages. In (6) we first performed the exploratory stage as we had to figure out the best AAM label.

+ __Solution-__ we can use the exploratory stage to choose the top 5-15 most influential variables out of 120+ variables (without using any permutation tests). Then we will run gen test only on the selected 5-15 variables. This will avoid the multiple comparisons problem when estimating the p-values.
This will also let us speak about other potential variables such as depression symptoms, video gaming, etc. from the explorative stage in the discussions.
We should also take care to not select variables that are approx the same from this stage, for example, Alc_lastmnt_TLFBFU3 and Alc_lastmnt_FU3, as this will be a waste. Let's decide these together.
See my analysis at [05_Roshan_final_analysis/Exploratory-stage-conclusions](#Exploratory-stage-conclusions) and [#Filter-selected-variables](#Filter-selected-variables).

__Minor pts (4) for chapter Introduction:__
+ (a) Can make it more concise after we finish all the other sections.
+ (b) Need to motivate why this follow-up study is important and what will be its contributions
+ (c) info about 3 data subsets can be removed completely.

__Major pts (5) for chapter Discussion:__

+ (a) Some variables (ex: Eating disorder, sensation seeking) survive permutation tests. Add a paragraph discussing the implications of this and how it relates to the literature. Some additional analysis would also be necessary here - such as what ratio of binge drinkers are also binge eaters? What sex are they predominantly, and so on.
+ (b) We should also talk about the neuroimaging side of this:
Discuss how the brain regions associated with BL are linked to the variables we found and binge drinking in the literature. If necessary, we can also generate the feature importance for *BL-Binge-0* (would be 2-3 days effort for me).
